packages/assets/styles/variables.scss

```diff
@@ -1,26 +1,110 @@
 .light-properties {
-	--color-bg: #ebebeb;
-	--color-raised-bg: #ffffff;
-	--color-super-raised-bg: #e9e9e9;
-	--color-button-bg: #f5f5f5;
+	--surface-l1: #ebebeb;
+	--surface-l2: #f5f5f5;
+	--surface-l3: #f8f8f8;
+	--surface-l4: #ffffff;
+	--surface-l5: #e6e6e6;
+
+	--color-red-50: #fef2f2;
+	--color-red-100: #fee5e7;
+	--color-red-200: #fccfd3;
+	--color-red-300: #faa7b1;
+	--color-red-400: #f67687;
+	--color-red-500: #ed4661;
+	--color-red-600: #cb2245;
+	--color-red-700: #b7193d;
+	--color-red-800: #9a1739;
+	--color-red-900: #841738;
+	--color-red-950: #490819;
+	--color-red: var(--color-red-600);
```

Comment:
feel like maybe these should be generated? it seems a little obnoxious if we ever adjust the colors or add a new theme to manually change a dozen variables.

Reply:
They are already generated using the base color

Reply:
See https://uicolors.app/generate/7616e6